harness_scripts/functional_by_framework/check_by_framework_run1_step_entrypoint.sh

```diff
@@ -26,22 +29,22 @@ pip uninstall datarobot-drum datarobot-mlops -y
 
 title "Installing dependencies, with datarobot-drum installed from source-code"
 
-pushd custom_model_runner
+cd "${ROOT_DIR}/custom_model_runner"
 
-if [ "$FRAMEWORK" = "java_codegen" ]; then
+if [ "${FRAMEWORK}" = "java_codegen" ]; then
     make java_components
 fi
 
 temp_requirements_file=$(mktemp)
-cp ${REQ_FILE_PATH} ${temp_requirements_file}
+cp ${ENV_REQ_FILE_PATH} ${temp_requirements_file}
 # remove DRUM from requirements file to be able to install it from source
 sed -i "s/^datarobot-drum.*//" ${temp_requirements_file}
 
-[ "$FRAMEWORK" = "r_lang" ] && EXTRA="[R]" || EXTRA=""
-pip install --force-reinstall -r ${temp_requirements_file} .$EXTRA
+[ "${FRAMEWORK}" = "r_lang" ] && EXTRA="[R]" || EXTRA=""
+pip install --force-reinstall -r ${temp_requirements_file} .${EXTRA}
```

Comment:
@zohar-mizrahi
 I wanted your opinion. 
 This script runs in the env image, which has all the requirements installed.
 Do we need to reinstall them, or  reinstalling only drum should be enough. Unless we hit the problem with `datarobot` / `mlops` imports.

Reply:
Well, probably it's redundant, although I used to benefit from it by using the `base` image to run this harness script. This block ensured me that all the dependencies are installed for that environment.